### PR TITLE
Query Browser: Add graph tooltip

### PR DESCRIPTION
**Pull request by kyoto**
Customize the tooltip layout using a SVG `<foreignObject>`.

Some styling is left for a follow up PR, including adding a tooltip arrow.

Also includes a bug fix for displaying a single query input by default.

screenshot

### CODE REVIEW

File: frontend/public/components/monitoring/_monitoring.scss [hunk 1]
```diff
@@ -130,6 +130,33 @@
   padding-right: 0;
 }
 
+.query-browser__tooltip {
+  background-color: #151515;
+  color: #eee;
+  overflow: scroll;
+  padding: 15px;
+  z-index: 999999;
```
Comment (spadgett) [inline frontend/public/components/monitoring/_monitoring.scss]: maybe

```suggestion
  z-index: $zindex-tooltip;
```

although there's probably a PF4 var that's better

cc @rhamilto

Comment (kyoto) [reply frontend/public/components/monitoring/_monitoring.scss]: https://github.com/openshift/console/pull/2101 resolves this by removing the `z-index`.

**Comment by kyoto** [comment]
Superseded by https://github.com/openshift/console/pull/2101, so closing this PR.

**Pull request closed without merging**